routes/transaction.go

```diff
@@ -804,16 +779,48 @@ func (fes *APIServer) BurnBitcoinStateless(ww http.ResponseWriter, req *http.Req
 
 		SerializedTxnHex:   hex.EncodeToString(bitcoinTxnBytes),
 		TxnHashHex:         bitcoinTxn.TxHash().String(),
-		BitCloutTxnHashHex: bitCloutTxnHashHex,
+		BitCloutTxnHashHex: bitcloutTxnHashString,
 
 		UnsignedHashes: unsignedHashes,
 	}
-	if err := json.NewEncoder(ww).Encode(res); err != nil {
+	if err = json.NewEncoder(ww).Encode(res); err != nil {
 		_AddBadRequestError(ww, fmt.Sprintf("BurnBitcoin: Problem encoding response as JSON: %v", err))
 		return
 	}
 }
 
+// GetNanosFromSats - convert Satoshis to BitClout nanos
+func (fes *APIServer) GetNanosFromSats(satoshis uint64, feeBasisPoints uint64) (uint64, error){
+	usdToBTC, err := GetUSDToBTCPrice()
+	if err != nil {
+		return 0, fmt.Errorf(" Problem getting usd to btc exchange rate: %v", err)
+	}
+	usdCentsPerBitcoin := usdToBTC * 100
+	usdCents := (float64(satoshis) * usdCentsPerBitcoin) / math.Pow(10, 8)
+	return fes.GetNanosFromUSDCents(usdCents, feeBasisPoints)
+}
+
+// GetNanosFromUSDCents - convert USD cents to BitClout nanos
+func (fes *APIServer) GetNanosFromUSDCents(usdCents float64, feeBasisPoints uint64) (uint64, error){
+	// Get Exchange Price gets the max of price from blockchain.com and the reserve price.
+	usdCentsPerBitClout, err := fes.GetExchangeBitCloutPrice()
+	if err != nil {
+		return 0, err
+	}
+	conversionRateAfterFee := float64(usdCentsPerBitClout) * ((100.0 + (float64(feeBasisPoints) / 100.0))/ 100.0)
```

Comment:
I think it would be less confusing to write

`float64(usdCentsPerBitClout) * (1+ (float64(feeBasisPoints) / 100.0))`

Reply:
that's technically incorrect as the fee is in basis points so I'm going to make it 
`float64(usdCentsPerBitClout) * (1 + (float64(feeBasisPoints) / (100.0 * 100.0)))`